docs/reference/r-releases/r-rev195_149.md

```diff
@@ -0,0 +1,108 @@
+# Revision 195/149
+<sub>Tue, Dec 10th, 2024</sub>
+
+Dear community,
+
+We are pleased to report that we have just released a new updated version for the Charmed Apache Kafka bundles on the `3/stable` channel, 
+upgrading Charmed Apache ZooKeeper revision from 136 to 149, and Charmed Apache Kafka from 156 to 195.
+
+The new release comes with a number of new features from the charms, from Juju secrets support, OAuth/OIDC authentication support, various improvements in the UI/UX and dependencies upgrades.  
+
+Please reach out should you have any question, comment, feedback or information. You can find us here in [Matrix](https://matrix.to/#/#charmhub-data-platform:ubuntu.com) or also on [Discourse](https://discourse.charmhub.io/).
+
+## Kafka
+
+New features and bug fixes in the Charmed Apache Kafka operator:
```

Comment:
Please, double-check that we want to talk specifically about the operator only here.